app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import { t } from "i18next";
+import { MenuHorizontal } from "neetoicons";
+import { Avatar, Typography, Dropdown } from "neetoui";
+import { noop } from "utils";
+
+const {
+  Menu,
+  MenuItem: { Button: MenuButton },
+} = Dropdown;
+
+export const getColumnData = setIsDeleteAlertOpen => [
```

Comment:
Here let's name this `buildColumnData` since we are passing in an argument and we are building a particular structure for the column data.